src/views/rebenchdb.R

```diff
@@ -57,7 +57,8 @@ main_data_from <- "
       JOIN Run ON runId = run.id
       JOIN Suite ON suiteId = suite.id
       JOIN Benchmark ON benchmarkId = benchmark.id
-      JOIN Executor ON execId = executor.id "
+      JOIN Executor ON execId = executor.id 
+      "
```

Comment:
Seems a non-functional, i.e. whitespace change, would be good to avoid if not needed or better in some way.